SignalsIntegrationTests/SignalsIntegrationTests/ZeroOrderPolicyTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿using Domain;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SignalsIntegrationTests.Infrastructure;
+using System;
```

Comment:
done.
still, it treats Microsoft.VisualStudio.TestTools.UnitTesting as non system using :(